docs/tools.rst

```diff
@@ -55,10 +58,43 @@ Example
 .. code-block:: bash
 
    nvidiaAIAAListModels \
-      -server http://10.110.45.66:5000/v1 \
+      -server http://0.0.0.0:5000 \
       -label spleen
 
 
+Session
+------------
+
+Provides implementation for ``nvidia::aiaa::Client::create_session()`` API.
+For more details refer `aiaa-session.cpp <https://github.com/NVIDIA/ai-assisted-annotation-client/blob/master/src/cpp-client/tools/aiaa/aiaa-session.cpp>`_
```

Comment:
links should be 
https://github.com/NVIDIA/ai-assisted-annotation-client/blob/master/cpp-client/tools/aiaa/aiaa-session.cpp
without "src"

Reply:
fixed